.gitignore

```diff
@@ -85,3 +85,6 @@ typings/
 
 # DynamoDB Local files
 .dynamodb/
+
+# Mac system files
+.DS_Store
```

Comment:
This should probably be ignored in your Git global ignore instead of checking into the repo. https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore